docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,13 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using MUI's products to get inspired for your next project.</p>
```

Comment:
```suggestion
<p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
```
To be consistent with that direction of "reflect reality". Currently, these are all apps using Material UI — as far as we know. Later on, I think we can have a more structured Case Study page where we dissect their experience using whatever MUI product. Does it make sense?

Reply:
Yeah that makes sense to me.